seonshine_frontend/src/components/molecules/accountVerification/accountVerificationLayout.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode } from 'react';
+
+import { Stack, Typography } from '@mui/material';
+
+type AccountVerificationLayoutProps = {
+  title?: string;
+  description: string;
+  children: ReactNode;
+};
+
+const AccountVerificationLayout: React.FC<AccountVerificationLayoutProps> = ({ title, description, children }) => {
```

Comment:
`React.FC` is the old way of defining a Function component. It's not even recommended. Don't need to fix, just note